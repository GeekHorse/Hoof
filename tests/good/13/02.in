new right one done
new in subonea done
new in subsubonea done
new out suboneb done
up
delete value
save
