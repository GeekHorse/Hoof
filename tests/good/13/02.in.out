new right one done
new in suboneb done
out
